ouroboros-consensus-cardano/tools/db-analyser/Block/Cardano.hs

```diff
@@ -111,6 +111,9 @@ mkCardanoProtocolInfo genesisByron signatureThreshold genesisShelley initialNonc
       ProtocolParamsMary {
           maryProtVer    = ProtVer 4 0
         }
+      ProtocolParamsAlonzo {
+          alonzoProtVer    = ProtVer 5 0
```

Comment:
This should eventually be 5, but I think we might want it to remain 4 for now. Duncan slated a change, hopefully for this sprint, that would let us handle this more smoothly (see #2938); we can discuss next time we chat.

Reply:
Ahh, wait. I got a little ahead of myself. I was thinking of the `cardano-node` repo, but this is `ouroboros-consensus-cardano:exe:db-analyser`. I'm not sure what we should do here -- continuing the existing pattern is probably fine for an internal tool.